gorm-rest/src/main/groovy/yakworks/rest/gorm/controller/RestRepoApiController.groovy

```diff
@@ -49,6 +48,7 @@ trait RestRepoApiController<D> extends RestApiController {
 
     //Need it to access log and still compile static in trait (See https://issues.apache.org/jira/browse/GROOVY-7439)
     //final private static Logger log = LoggerFactory.getLogger(this.class)
+    @SuppressWarnings(['LoggerWithWrongModifiers'])
```

Comment:
why do we need this? shouldnt we be setting it up with proper modifiers? put comment on SuppressWarnings if we really need it.

Reply:
there's private static final line above it and its commented, so i thought u did this intentionally @basejump